remix-solidity/src/compiler/compiler-worker.ts

```diff
@@ -2,8 +2,8 @@
 
 var solc = require('solc/wrapper')
 
-var compileJSON = function () { return '' }
-var missingInputs = []
+var compileJSON: any|null = (input) => { return '' }
```

Comment:
What is the real type of compileJSON ? 
What about `const compileJSON: (input: SomeType) => string` ?

Reply:
updated